frontend/app/views/giraffe/contribute.scala.html

```diff
@@ -60,7 +60,7 @@ <h3 class="form__column-heading form__column-heading--first headline-text--bolde
             <div class="form__column">
                 <h3 class="form__column-heading headline-text--bolder"><span class="num">2</span> Your amount</h3>
                 <ul class="u-unstyled">
-                    <li id="currency_field" class="form-field">
+                    <li id="currency_field" class="form-field" hidden>
```

Comment:
yes, if we can imagine it being turned on/off via JavaScript - but this was simply a shortcut to "disable" this functionality altogether so I think it's fine as an attribute here